README.md

```diff
@@ -24,6 +24,7 @@ and strongly encouraged.
 ### Sales and Marketing
 
  - [Sales](./sales)
+ - [Content](./content)
```

Comment:
Can we call it `marketing`?

Reply:
I would vote for "Marketing & Communication"

Reply:
I'm with @joepavitt, I'll edit the PR.

Reply:
Updated, OK now?

Reply:
✔️